lib/split/experiment.rb

```diff
@@ -225,10 +236,10 @@ def resettable?
 
     def reset
       Split.configuration.on_before_experiment_reset.call(self)
-      alternatives.each(&:reset)
       reset_winner
       Split.configuration.on_experiment_reset.call(self)
       increment_version
+      alternatives.each(&:reset)
```

Comment:
I'm guessing this is moved because resetting the alternatives necessitates the new version. Is that correct?

Did we check if there's any functionality which relies on the `on_experiment_reset` hook which might be affected by this change?

Reply:
Will make a note to do so

Reply:
a scan of our primary app does not show us using this callback.

Reply:
Cool. In that case should we remove it? Or maybe replace it with a log message saying it's not supported in our forked message? In case someone does try to use it in the future.

Reply:
I was just pointing out that we don't use it. People should still be able to piggyback off it.